### PR TITLE
fix: liveness probe command

### PR DESCRIPTION
**Pull request by atobaum**
**Description of the change**
Added `node` to liveness probe.

**Context**
Due to `node` command being omitted in the liveness probe of deployment, the following events are occurring in my k8s cluster:
```
Name:             uptime-kuma-5b765cc74c-7tfvk.17cb9700282d4285
Namespace:        uptime-kuma-system
Labels:           <none>
Annotations:      <none>
API Version:      v1
Count:            363130
Event Time:       <nil>
First Timestamp:  2024-05-02T06:08:30Z
Involved Object:
  API Version:       v1
  Field Path:        spec.containers{uptime-kuma}
  Kind:              Pod
  Name:              uptime-kuma-5b765cc74c-7tfvk
  Namespace:         uptime-kuma-system
  Resource Version:  42515422
  UID:               d2200d3b-0bfd-42d3-a8d2-d6917ae31d03
Kind:                Event
Last Timestamp:      2024-06-13T06:56:48Z
Message:             (combined from similar events): Liveness probe errored: rpc error: code = Unknown desc = failed to exec in container: failed to start exec "bb64cb3256790c20687b7b119ed4dac6fa5705bc736d691628d6f8acc937e917": OCI runtime exec failed: exec failed: unable to start container process: exec: "extra/healthcheck": stat extra/healthcheck: no such file or directory: unknown
Metadata:
  Creation Timestamp:  2024-05-02T06:08:30Z
  Resource Version:    98857066
  UID:                 48cd7e56-ec26-4289-ac4b-9730a9572a4c
Reason:                Unhealthy
Reporting Component:   
Reporting Instance:    
Source:
  Component:  kubelet
  Host:       standard-nodes-w-1rrt
Type:         Warning
Events:       <none>
```

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/uptime-kuma/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "1.23.13"
+appVersion: "1.23.14"
```
Comment (dirsigler) [inline charts/uptime-kuma/Chart.yaml:2]: `appVersion` reflects the version of the UptimeKuma Image used. So please keep it at `1.23.13`.

**Review by dirsigler — CHANGES_REQUESTED** [review]
Please also bump the Chart Version. Would recommend `2.19.0`.
You can also run [helm-docs](https://github.com/norwoodj/helm-docs) afterwards to update the README.md

File: charts/uptime-kuma/Chart.yaml (hunk 1 above)
Comment (atobaum) [reply charts/uptime-kuma/Chart.yaml]: I applied your comment. thank you.

**Review by dirsigler — APPROVED** [review]
lgtm

**Comment by dirsigler** [comment]
Thank you very much @atobaum for the work! ❤️

**Comment by dirsigler** [comment]
Just as an FYI @atobaum , I "reverted" the change again due to the "node" command being deprecated (see [this code](https://github.com/louislam/uptime-kuma/blob/ae224f9e188b1fc32ed8729818710975589cdce7/extra/healthcheck.js#L6) ).

But now I also provided the functionality to configure the livenessProbe as desired, so if you encounter any issues and still need to use the "node" command, you can add it to your release `values.yaml`.

**Pull request merged**